contracts/upgrader/src/contract.rs

```diff
@@ -0,0 +1,82 @@
+use crate::error::ContractError;
+use axelar_soroban_std::traits::ThenOk;
+use soroban_sdk::{contract, contractimpl, Address, BytesN, Env, String, Symbol, Val};
+
+const MIGRATE: &'static str = "migrate";
+const UPGRADE: &'static str = "upgrade";
+const VERSION: &'static str = "version";
```

Comment:
not sure there is a lot of benefit to it and will lead to unnecessary dependencies. In my opinion the contracts don't need to know that there is an upgrader contract calling them

Reply:
This interface only defines that the contract is upgradable, not that there is an upgrader contract. We can put this interface in the axelar-soroban-std instead, and the upgrader can use it instead of hardcoding function interfaces